js_src/react_dom_dev.js

```diff
@@ -0,0 +1,10 @@
+const ShallowRenderer = require('react-test-renderer/shallow');
+const TestUtils = require('react-dom/test-utils');
+
+
+if (!window.React.addons) {
+    window.React.addons = {};
+}
+window.React.addons.TestUtils = TestUtils;
+window.ReactShallowRenderer = ShallowRenderer;
```

Comment:
Should we preserve this path under test utils? 🤷‍♂️

Reply:
Hmm nice catch, ill look into this.

Reply:
Do we still need this as a top-level export as well?

Reply:
apparently not.